news/5128.update.rst

```diff
@@ -0,0 +1 @@
+Updated a hook for sqlalchemy, by adding a hidden import.
```

Comment:
`update` isn't a valid type of entry. I'm fairly certain this should be named `news/5128.hooks.rst`. @Legorooj Can you confirm this?

Reply:
Confirmed. Didn't spot that - good thing we have that check eh?

Reply:
By the way - I named this file after reading this line **'For example, :file:`news/42.feature.rst` for a new feature that is proposed in pull request #42.'** on this page https://github.com/pyinstaller/pyinstaller/tree/develop/news

Perhaps it could be edited to be clearer? 

It conflicts with the advice at https://github.com/pyinstaller/pyinstaller-hooks-contrib/blob/master/news/README.txt, but I took the advice at  https://github.com/pyinstaller/pyinstaller/tree/develop/news given I was saving the file in Pyinstaller/pyinstaller/news/